TypeCobol.Test/Parser/Programs/Cobol85/ExecSql/ExecSqlWithCommit.rdz-Tokens.txt

```diff
@@ -1,4 +1,4 @@
-Line 10[12,19] <37, Warning, General> - Warning: a End statement is not aligned with the matching opening statement
+Line 5[31,39] <27, Error, Syntax> - Syntax error : extraneous input 'CUSTOMERS' expecting {separator, numeric literal, character string, symbol, statement starting keyword, keyword, Formalized Comments elements, Sql statement starting keyword}
```

Comment:
That should be ok after merging #2189, right ?

Reply:
yes